### DISCUSSION TITLE
List of projects using Micronaut

### DISCUSSION THREAD

**Discussion by debuglevel** [micronaut-projects/micronaut-core#5149] [Q&A]
Hello there,

do you know a list of "real" projects on GitHub or GitLab that are using Micronaut? Or some useful tool for filtering/searching on GitHub to find such projects? Because I would really like to read some more code and look how other developers are working with Micronaut, but most times I just find some demo projects or tutorials which are not as interesting as real projects. :-)

Best regards and hope that *Discussions* is the right place to ask such questions.

**Comment by saw303** [comment]
As a starting point you could looking for Java source code that uses `import io.micronaut.http.annotation.Controller;`. This will find you all Java source code that contains a Micronaut controller. 

https://github.com/search?q=import+io.micronaut.http.annotation.Controller%3B&type=code
